cmake/boost.cmake

```diff
@@ -11,7 +11,8 @@ ExternalProject_Add(ppc_boost
         BINARY_DIR        "${CMAKE_CURRENT_BINARY_DIR}/ppc_boost/build"
         INSTALL_DIR       "${CMAKE_CURRENT_BINARY_DIR}/ppc_boost/install"
         CONFIGURE_COMMAND "${CMAKE_COMMAND}" -S "${CMAKE_SOURCE_DIR}/3rdparty/boost/" -B "${CMAKE_CURRENT_BINARY_DIR}/ppc_boost/build/"
-        -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER} -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER} -G${CMAKE_GENERATOR} -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DBOOST_ENABLE_MPI=ON
+        -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER} -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER} -G${CMAKE_GENERATOR} -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        -DBOOST_INCLUDE_LIBRARIES=mpi -DBOOST_ENABLE_MPI=ON
```

Comment:
Potential problem with functions required for the following tasks.

TODO: check